### PR TITLE
Update serving section in the homepage

### PR DESCRIPTION
**Pull request by yubozhao**
Fix grammar for open-source and added BentoML to the serving section in the website.

### CODE REVIEW

**Review by terrytangyuan — COMMENTED** [review]
Thanks!

/lgtm

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: content/en/_index.html [hunk 1]
```diff
@@ -129,15 +131,15 @@ <h5 class="card-title text-white section-head">Pipelines</h5>
...
+			<a target="_blank" rel="noopener" href="/docs/components/training/mxnet/"> Apache MXNet</a>,
 		  	<a target="_blank" rel="noopener" href="/docs/components/training/mpi/"> MPI</a>,
 		  	<a target="_blank" rel="noopener" href="/docs/components/training/xgboost/"> XGBoost</a>,
 			<a target="_blank" rel="noopener" href="/docs/components/training/chainer/"> Chainer</a>, and more. We
-			also integrate with 
-			<a target="_blank" rel="noopener" href="https://istio.io/">Istio</a> and 
-			<a target="_blank" rel="noopener" href="https://www.getambassador.io/">Ambassador</a> for ingress, 
+			also integrate with
+			<a target="_blank" rel="noopener" href="https://istio.io/">Istio</a> and
+			<a target="_blank" rel="noopener" href="https://www.getambassador.io/">Ambassador</a> for ingress,
```
Comment (jbottum) [inline content/en/_index.html:142]: is ambassador being supported in 1.6?

File: content/en/_index.html [hunk 2]
```diff
@@ -129,15 +131,15 @@ <h5 class="card-title text-white section-head">Pipelines</h5>
...
 			<a target="_blank" rel="noopener" href="https://nuclio.io/">Nuclio</a> as a
```
Comment (jbottum) [inline content/en/_index.html:143]: Is the nuclio integration actively supported in 1.6 ?

File: content/en/_index.html [hunk 3]
```diff
@@ -129,15 +131,15 @@ <h5 class="card-title text-white section-head">Pipelines</h5>
...
             fast multi-purpose serverless framework, and <a target="_blank" rel="noopener" href="https://www.pachyderm.io/">Pachyderm</a> for managing
             your data science pipelines.
```
Comment (jbottum) [inline content/en/_index.html:145]: is there an integration definition or plan for Pachyderm in 1.6 ?

File: content/en/_index.html [hunk 4]
```diff
@@ -93,7 +93,9 @@ <h5 class="card-title text-white section-head">Model serving</h5>
             TensorFlow models to Kubernetes. Kubeflow is also integrated with
             <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/seldon/">Seldon Core</a>, an open source platform for deploying machine learning
             models on Kubernetes, <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/tritoninferenceserver/">NVIDIA Triton Inference Server</a> for
-            maximized GPU utilization when deploying ML/DL models at scale, and <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/mlrun/">MLRun Serving</a>, an open-source serverless framework for deployment and monitoring of real-time ML/DL pipelines.
+            maximized GPU utilization when deploying ML/DL models at scale, <a target="_blank" rel="noopener" href="/">BentoML</a>, an open-source platform for building,
```
Comment (kimwnasptd) [inline content/en/_index.html:96]: @yubozhao how is BentoML integrated with `Model serving` in Kubeflow? Is this via KServe? Is there some documentation for it?

File: content/en/_index.html (hunk 1 above)
Comment (jbottum) [reply content/en/_index.html:142]: let's address this in another PR

File: content/en/_index.html (hunk 2 above)
Comment (jbottum) [reply content/en/_index.html:143]: let's address this in another pr

File: content/en/_index.html (hunk 3 above)
Comment (jbottum) [reply content/en/_index.html:145]: let's address this in another pr

File: content/en/_index.html (hunk 4 above)
Comment (jbottum) [reply content/en/_index.html:96]: @yubozhao I agree with Kimonas' questions above.    To post this update, Bento should have some docs on how users can execute a workflow from Kubeflow to Bento.

**Comment by varodrig** [comment]
It seems this content is not longer valid. I recommend to close this PR.
@terrytangyuan @yubozhao @Arhell

**Pull request closed without merging**